packages/svelte/types/index.d.ts

```diff
@@ -484,6 +484,12 @@ declare module 'svelte' {
 	 * ```
 	 * */
 	export function untrack<T>(fn: () => T): T;
+	/**
+	 * When used inside a [`$derived`](https://svelte.dev/docs/svelte/$derived),
+	 * any state updates to state is allowed.
```

Comment:
It's called `unsafe` and this explanation leaves me wondering how to ensure that my usages of it are safe in practice. Maybe the idea is that if you use it, you should already understand it. However maybe there could be a bit lengthier explanation?

Reply:
There'd probably be a more detailed description in the online documentation, probably in the `$derived` page